vendor.conf

```diff
@@ -83,6 +85,7 @@ github.com/xeipuuv/gojsonreference bd5ef7bd5415a7ac448318e64f11a24cd21e594b
 github.com/xeipuuv/gojsonschema 93e72a773fade158921402d6a24c819b48aba29d
 golang.org/x/crypto 0709b304e793a5edb4a2c0145f281ecdc20838a4
 golang.org/x/net a680a1efc54dd51c040b3b5ce4939ea3cf2ea0d1
+golang.org/x/oauth2 d668ce993890a79bda886613ee587a69dd5da7a6
```

Comment:
Perhaps we should make vendoring match the upstream kubernetes-on-compose version? https://github.com/docker/compose-on-kubernetes/blob/939a5ba6c6dbfc78e6cc29b612716d5688b9a323/Gopkg.lock#L725

That way we know we're using the same version as the tests on kubernetes-on-compose run on


Might be worth checking if kubernetes-on-compose is using more recent versions of other packages as well